### DISCUSSION TITLE
Upload of files and deposit of records in parallel?

### DISCUSSION THREAD

**Discussion by rkrug** [eblondel/zen4R#112] [General]
Hi

I have a large number (>100) rather large (~~~30GB) deposits. I am thinking - is zen4R ahead safe, and are there any problems from Zenodo side with depositing records and uploading files in parallel? Any experiences?

Thanks,

Rainer

**Comment by eblondel** [comment]
You may need to ask this to Zenodo support team (i'm not part of it). zen4R is an initiative outside Zenodo, not funded nor supported by Zenodo, so as for any FOSS, no waranty :-). This said, zen4R uses standards client methods to interface the Zenodo API, and the methods designed are tested through integration tests. At the end zen4R is **just** a R client to their API, so any issue related to availability and capacities would not be to deal at zen4R level but at Zenodo level (server-side).

Personnally, I had experimented issues when trying to use the Zenodo API (with zen4R) to upload very large data files, including that led to unavailability of the Zenodo e-infrastructure at some point.

**Reply by rkrug** [reply]
Thank - I did not expect you to answer on behalf of Zenodo - but maybe you  had heard / experienced something. 

Ups - I will try it anyway (as I am to lazy to deposit more than 130 large files by hand...) and let you know about my experiences. 

Thanks,

Rainer

**Reply by eblondel** [reply]
Yes indeed i'm interested to know how it will work with large files.
